book/src/SUMMARY.md

```diff
@@ -4,6 +4,7 @@
 
 - [Installation](installation.md)
 - [Usage](usage.md)
+    - [Extending Clippy coverage](lib_usage.md)
```

Comment:
I would make this it's own section, as it is not really how to use clippy (as a tool), but more about how create authors can also benefit from some clippy lints by adding attributes.

Section title(maybe) : Attributes for Crate Authors

Reply:
done, thx